### ISSUE TITLE
"Sign in to MAS" task fails with Ansible 2.8

### ISSUE THREAD

**Issue by jklaiho** [geerlingguy/ansible-role-mas#19]
I've specified `mas_email` and `mas_password` in my configuration as follows:

```
mas_email: redacted@redacted.com
mas_password: !vault |
          $ANSIBLE_VAULT;1.1;AES256 (...etc...)
```

This causes the "Sign in to MAS when email and password are provided" task to be run, but fail. Two things happen:

First, we get this:

```
[DEPRECATION WARNING]: evaluating redacted@redacted.com as a bare variable, this behaviour will go away and you might need to add |bool to the 
expression in the future. Also see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in version 2.12. Deprecation 
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

This is a result of the task doing `when` checks against `mas_email` and `mas_password` without the `|bool` cast, obviously. But more seriously, this happens:

```
FAILED! => {"msg": "The conditional check 'mas_email' failed. The error was: template error while templating string: unexpected char '@' at 14. String: {% if redacted@redacted.com %} True {% else %} False {% endif %}

The error appears to be in '/path/to/playbookdir/roles/geerlingguy.mas/tasks/main.yml': line 11, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Sign in to MAS when email and password are provided.
  ^ here
"}
```

So, despite Ansible 2.8 saying it has merely deprecated the old behavior, it seems to have also changed enough to actually break in this case.

**Comment by jklaiho** [comment]
Tested it now; fixed by simply adding `|bool` to each instance of `mas_email` and `mas_password` in the `when:` clauses of tasks/main.yml.

**Comment by geerlingguy** [comment]
Thanks for reporting this! I believe it should be fixed in 1.3.1 / with the above commit.
